documentation/configuration_files/service.md

```diff
@@ -41,3 +41,36 @@ dependency-data:
 
 * `remote.cpu`: Number of CPU units to reserve for service container (see "cpu" under [Container Definitions/Environment](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/task_definition_parameters.html#container_definition_environment))
 * `remote.memeory`: The hard limit (in MiB) of memory to allocate to the service container (see "memory" under [Container Definitions](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/task_definition_parameters.html#container_definitions))
+
+# Service types
+
+## Public services
+Public services expose an external and internal http endpoint. Container ports must be listed in `development.port` and `production.port`.
+
+Endpoints available in local development:
+  - `#{SERVICE_ROLE}_EXTERNAL_ORIGIN: http://localhost:#{host-port}`: The external endpoint at which a public service can be reached. Exosphere automatically picks an available host port and binds it to the specified service container port.
```

Comment:
Thoughts on updating these to something like 

> `#{SERVICE_ROLE}_EXTERNAL_ORIGIN`: The external endpoint ... 

I don't think we need to give the user the implementation details here. We should probably mention all of there *_ORIGIN include the http or https protocol but I don't think we need anything beyond that. They can look at the docker-compose file or the terraform var files to see the values